dali/base/dautils.cpp

```diff
@@ -924,6 +924,36 @@ StringBuffer &CDfsLogicalFileName::makeXPathLName(StringBuffer &lfnNodeName) con
     return lfnNodeName;
 }
 
+static bool begins(const char *&ln,const char *pat)
```

Comment:
Possibly, but my main objection was that there serveral (2 perhaps) in existing code already and this added a 3rd copy of the same code.
if dali + tools are the only current uses, it can be an exported method of dautils.cpp - and the other copies can be removed.